packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -265,11 +269,45 @@ const PRESETS = [
             ['project.commerce.shortCode']: 'kv7kzm78',
             ['project.commerce.isSlasPrivate']: false,
             ['project.einstein.clientId']: '1ea06c6e-c936-4324-bcf0-fada93f83bb1',
-            ['project.einstein.siteId']: 'aaij-MobileFirst'
+            ['project.einstein.siteId']: 'aaij-MobileFirst',
+            ['project.demo.enableDemoSettings']: false
         },
         assets: ['translations'],
         private: false
     },
+    {
+        id: 'retail-react-app-demo-private',
+        name: 'Retail React App Demo Private Client',
+        description: `
+            Generates a project using the settings for a special B2C Commerce
+            instance that is used for demo purposes and sets it up with a private SLAS client.
+
+            This has social and passwordless login enabled but requires a client secret to run.
```

Comment:
In the future, we may have new features that are enabled for only the demo site. That's the goal of the new preset, is that right? If so, perhaps make it clear that it's not just for the social/passwordless logins.